scripts/clickhouse_operator_sync.sh

```diff
@@ -18,13 +18,13 @@ TMP_FOLDER="$(mktemp -d)"
 trap 'rm -rf -- "$TMP_FOLDER"' EXIT
 
 CLICKHOUSE_OPERATOR_TAG="0.18.4"
-URL="https://raw.githubusercontent.com/Altinity/clickhouse-operator/${CLICKHOUSE_OPERATOR_TAG}/deploy/operator/clickhouse-operator-install-template.yaml"
+REPO_URL="https://github.com/Altinity/clickhouse-operator.git"
```

Comment:
Why is this needed?

Reply:
Because running `cat-clickhouse-operator-install-yaml.sh` requires multiple template files, so instead of downloading all of them separately, it's easier to just clone the repo and have everything that is required there.